src/lib/TemplateManager.ts

```diff
@@ -176,10 +180,33 @@ export class TemplateManager {
         .map(async (file: string) => {
           await fs.copy(
             path.join(this.templatesDir, "commands", file),
-            path.join(commandTargetDir, file)
+            path.join(commandTargetDir, file),
           );
           copiedFiles.push(path.join(commandTargetDir, file));
-        })
+        }),
     );
   }
+
+  private async copyRooSpecificFiles(copiedFiles: string[]): Promise<void> {
```

Comment:
Why don't we use https://github.com/codeaholicguy/ai-devkit/blob/main/src/util/env.ts#L64?